create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/PublicationConverterTest.java

```diff
@@ -0,0 +1,12 @@
+package no.sikt.nva.scopus;
+
+import org.junit.jupiter.api.Test;
+
+
+class PublicationConverterTest {
```

Comment:
Test is unnecessary as it is tested by the handler

Reply:
It is just a placeholder since the functionality will be implemented later. And we had to give JaCoco a bone to chew on.

Reply:
We will follow your test premisses.